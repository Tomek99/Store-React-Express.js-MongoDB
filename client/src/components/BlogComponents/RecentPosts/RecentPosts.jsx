import React from "react";
import styles from "./RecentPosts.module.scss";
import UserPost from "../../UserPost/UserPost";

function RecentPosts() {
  return (
    <div className={styles.RecentPosts}>
      <h1 className={styles.postsHeader}>Recent posts</h1>
      <div className={styles.divPosts}>
        <UserPost />
        <UserPost />
        <UserPost />
      </div>
    </div>
  );
}

export default RecentPosts;
