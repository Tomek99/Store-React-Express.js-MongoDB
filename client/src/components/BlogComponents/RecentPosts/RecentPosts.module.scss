.RecentPosts {
  color: var(--text-color);
  background-color: var(--light-greay);
  padding: 20px 0 20px 15px;
  border-radius: 12px;

  .postsHeader {
    font-size: 1.5rem;
    padding-bottom: 5px;
    margin-top: 10px;
    margin-bottom: 25px;
    border-bottom: var(--border);
  }
  .divPosts {
    display: flex;
    flex-direction: column;
    gap: 20px;
  }
}
