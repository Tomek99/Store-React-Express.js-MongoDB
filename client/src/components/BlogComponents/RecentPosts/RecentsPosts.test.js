import { render, screen } from "@testing-library/react";
import RecentPosts from "./RecentPosts";

test("should render component", () => {
  const { container } = render(<RecentPosts />);

  expect(container).toMatchSnapshot();
});
