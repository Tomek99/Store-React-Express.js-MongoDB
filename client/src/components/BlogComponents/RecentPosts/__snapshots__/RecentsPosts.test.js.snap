// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component 1`] = `
<div>
  <div
    class="RecentPosts"
  >
    <h1
      class="postsHeader"
    >
      Recent posts
    </h1>
    <div
      class="divPosts"
    >
      <div
        class="UserPost"
      >
        <img
          alt="userImage"
          class="userImage"
          src="https://i.pinimg.com/originals/ae/ec/c2/aeecc22a67dac7987a80ac0724658493.jpg"
        />
        <div
          class="userContent"
        >
          <p
            class="postData"
          >
            06.03.2023
          </p>
          <p
            class="userText"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </p>
        </div>
      </div>
      <div
        class="UserPost"
      >
        <img
          alt="userImage"
          class="userImage"
          src="https://i.pinimg.com/originals/ae/ec/c2/aeecc22a67dac7987a80ac0724658493.jpg"
        />
        <div
          class="userContent"
        >
          <p
            class="postData"
          >
            06.03.2023
          </p>
          <p
            class="userText"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </p>
        </div>
      </div>
      <div
        class="UserPost"
      >
        <img
          alt="userImage"
          class="userImage"
          src="https://i.pinimg.com/originals/ae/ec/c2/aeecc22a67dac7987a80ac0724658493.jpg"
        />
        <div
          class="userContent"
        >
          <p
            class="postData"
          >
            06.03.2023
          </p>
          <p
            class="userText"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
